config/toml.go

```diff
@@ -276,6 +276,31 @@ laddr = "{{ .GRPC.ListenAddress }}"
 [grpc.version_service]
 enabled = {{ .GRPC.VersionService.Enabled }}
 
+#
+# Configuration for privileged gRPC endpoints, which should **never** be exposed
+# to the public internet.
+#
+[grpc.privileged]
+# The host/port on which to expose privileged gRPC endpoints.
+laddr = "{{ .GRPC.Privileged.ListenAddress }}"
+
+#
+# Configuration specifically for the gRPC pruning service, which is considered a
+# privileged service.
+#
+[grpc.privileged.pruning_service]
+
+# Only controls whether the pruning service is accessible via the gRPC API - not
+# whether a previously set pruning service retain height is honoured by the
```

Comment:
@thanethomson Your English is not acceptable by the linter :stuck_out_tongue_closed_eyes: